docs/apache-airflow/concepts/dags.rst

```diff
@@ -713,7 +713,8 @@ Additional difficulty is that one DAG could wait for or trigger several runs of
 with different data intervals. The **Dag Dependencies** view
 ``Menu -> Browse -> DAG Dependencies`` helps visualize dependencies between DAGs. The dependencies
 are calculated by the scheduler during DAG serialization and the webserver uses them to build
-the dependency graph.
+the dependency graph. Note that if a DAG depends on another DAG which does not exist (for example a deleted DAG),
+the webserver will fail to render the dependency graph.
```

Comment:
mmm I don't think this is something for documentation. it sounds like a bug.
The webserver shouldn't fail - it should explain to the user why the page isn't being rendered

Reply:
Agreed.

Reply:
cc @bbovenzi  wdyt?

Reply:
@lidalei would you like to fix the bug?

Reply:
Yeh. Agree fixing it would be best

Reply:
I would like to. But it will take me quite some time to understand the code and create a PR. Considering the severity of the issue, I guess that's fine.

Reply:
This is not something for documentation either way.
A user that experience this problem won't go to the documentation to find why the page shows error.

@bbovenzi Maybe you can help with creating a proper error message in the UI for this case?

Reply:
Sure, just on the page itself if it fails to render?

Reply:
Is it possible to show an error message on the specific dag_ids that are missing (but still showing the page properly for all other dags)?